packages/swagger-ui/src/swagger/renderer.ts

```diff
@@ -37,6 +39,7 @@ const RENDER_TYPE_MAP = {
   operationsSorter: RENDER_TYPE.RAW,
   requestInterceptor: RENDER_TYPE.RAW,
   responseInterceptor: RENDER_TYPE.RAW,
+  persistAuthorization: RENDER_TYPE.BOOLEAN,
```

Comment:
I don't think it is necessary to add `RENDER_TYPE.BOOELAN` because you can just use `RENDER_TYPE.RAW` for boolean.

Reply:
yeah that'd be better.